app/views/meals/import/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for(:title) { 'Import Meal' } %>
+
+<h2>How To Import a CSV File</h2>
+<p>
+  You create a CSV file with the following as headers:
+  <ul>
+    <li>Date/Time (required, any parseable datetime string)</li>
+    <li>Location(s) (required, semicolon separated, location full name)</li>
+    <li>Capacity (optional, integer)</li>
+    <li>Formula (optional, formula name)</li>
+    <li>Head Cook (optional, person full name, email, or alternate ID (prefixed with "alt:"))</li>
+    <li>Extra roles (optional, One column for each extra role, optional, semicolon separated, people full names)</li>
+  </ul>
```

Comment:
I'm not sure this text will be necessary. Right now it reads a little too technical (e.g. "parseable") and I don't think folks will know what some of the stuff is. If we have a good example spreadsheet that should be enough I think.

So perhaps we could replace this with a few sentences explaining how to download the example file and what to do with it.